### PR TITLE
fix MaxResponseHeadersLength tests

### LINKED ISSUE #73930: Failures in SocketsHttpHandler_HttpClientHandler_MaxResponseHeadersLength_Http3
Full log: https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-72145-merge-c270b69ee20f4e269e/System.Net.Http.Functional.Tests/3/console.142a186b.log?helixlogtype=result

Hit in https://github.com/dotnet/runtime/pull/72145/checks?check_run_id=7832255512

```
System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_MaxResponseHeadersLength_Http3.LargeSingleHeader_ThrowsException(maxResponseHeadersLength: 15) [FAIL]
      System.Net.Quic.QuicException : Stream aborted by peer (268).
      Stack Trace:
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ResettableValueTaskSource.cs(157,0): at System.Net.Quic.ResettableValueTaskSource.TryComplete(Exception exception, Boolean final)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ResettableValueTaskSource.cs(221,0): at System.Net.Quic.ResettableValueTaskSource.TrySetException(Exception exception, Boolean final)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs(516,0): at System.Net.Quic.QuicStream.HandleEventPeerReceiveAborted(_PEER_RECEIVE_ABORTED_e__Struct& data)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs(570,0): at System.Net.Quic.QuicStream.HandleStreamEvent(QUIC_STREAM_EVENT& streamEvent)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs(601,0): at System.Net.Quic.QuicStream.NativeCallback(QUIC_HANDLE* connection, Void* context, QUIC_STREAM_EVENT* streamEvent)
        --- End of stack trace from previous location ---
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ResettableValueTaskSource.cs(267,0): at System.Net.Quic.ResettableValueTaskSource.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(168,0): at System.Net.Test.Common.Http3LoopbackStream.SendFrameAsync(Int64 frameType, ReadOnlyMemory`1 framePayload)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(123,0): at System.Net.Test.Common.Http3LoopbackStream.SendHeadersFrameAsync(Nullable`1 statusCode, IEnumerable`1 headers, Boolean qpackEncodeStatus)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(264,0): at System.Net.Test.Common.Http3LoopbackStream.SendResponseHeadersAsync(Nullable`1 statusCode, IEnumerable`1 headers)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(247,0): at System.Net.Test.Common.Http3LoopbackStream.SendResponseAsync(HttpStatusCode statusCode, IList`1 headers, String content, Boolean isFinal)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs(241,0): at System.Net.Test.Common.Http3LoopbackConnection.HandleRequestAsync(HttpStatusCode statusCode, IList`1 headers, String content)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs(93,0): at System.Net.Test.Common.Http3LoopbackServer.HandleRequestAsync(HttpStatusCode statusCode, IList`1 headers, String content)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs(93,0): at System.Net.Test.Common.Http3LoopbackServer.HandleRequestAsync(HttpStatusCode statusCode, IList`1 headers, String content)
        /_/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs(84,0): at System.Net.Http.Functional.Tests.HttpClientHandler_MaxResponseHeadersLength_Test.<>c__DisplayClass4_0.<<LargeSingleHeader_ThrowsException>b__1>d.MoveNext()
        --- End of stack trace from previous location ---
        /_/src/libraries/Common/tests/System/Threading/Tasks/TaskTimeoutExtensions.cs(120,0): at System.Threading.Tasks.TaskTimeoutExtensions.GetRealException(Task task)
        --- End of stack trace from previous location ---
        /_/src/libraries/Common/tests/System/Threading/Tasks/TaskTimeoutExtensions.cs(90,0): at System.Threading.Tasks.TaskTimeoutExtensions.WhenAllOrAnyFailed(Task[] tasks)
        /_/src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs(38,0): at System.Net.Test.Common.LoopbackServerFactory.<>c__DisplayClass5_0.<<CreateClientAndServerAsync>b__0>d.MoveNext()
        --- End of stack trace from previous location ---
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs(111,0): at System.Net.Test.Common.Http3LoopbackServerFactory.CreateServerAsync(Func`3 funcAsync, Int32 millisecondsTimeout, GenericLoopbackOptions options)
        /_/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs(72,0): at System.Net.Http.Functional.Tests.HttpClientHandler_MaxResponseHeadersLength_Test.LargeSingleHeader_ThrowsException(Int32 maxResponseHeadersLength)
        --- End of stack trace from previous location ---
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.Credentials_DomainJoinedServerUsesKerberos_UseIpAddressAndHostHeader_Success [SKIP]
      Condition(s) not met: "IsDomainJoinedServerAvailable"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.Proxy_DomainJoinedProxyServerUsesKerberos_Success [SKIP]
      Condition(s) not met: "IsDomainJoinedServerAvailable"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_Authentication_Test.Credentials_ServerChallengesWithWindowsAuth_ClientSendsWindowsAuthHeader [SKIP]
      Condition(s) not met: "IsNtlmInstalled"
    System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_MaxResponseHeadersLength_Http3.ThresholdExceeded_ThrowsException(maxResponseHeadersLength: 1, headersLengthEstimate: 1024) [FAIL]
      System.Net.Quic.QuicException : Stream aborted by peer (268).
      Stack Trace:
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ResettableValueTaskSource.cs(157,0): at System.Net.Quic.ResettableValueTaskSource.TryComplete(Exception exception, Boolean final)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ResettableValueTaskSource.cs(221,0): at System.Net.Quic.ResettableValueTaskSource.TrySetException(Exception exception, Boolean final)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs(516,0): at System.Net.Quic.QuicStream.HandleEventPeerReceiveAborted(_PEER_RECEIVE_ABORTED_e__Struct& data)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs(570,0): at System.Net.Quic.QuicStream.HandleStreamEvent(QUIC_STREAM_EVENT& streamEvent)
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs(601,0): at System.Net.Quic.QuicStream.NativeCallback(QUIC_HANDLE* connection, Void* context, QUIC_STREAM_EVENT* streamEvent)
        --- End of stack trace from previous location ---
        /_/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/ResettableValueTaskSource.cs(267,0): at System.Net.Quic.ResettableValueTaskSource.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(168,0): at System.Net.Test.Common.Http3LoopbackStream.SendFrameAsync(Int64 frameType, ReadOnlyMemory`1 framePayload)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(123,0): at System.Net.Test.Common.Http3LoopbackStream.SendHeadersFrameAsync(Nullable`1 statusCode, IEnumerable`1 headers, Boolean qpackEncodeStatus)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(264,0): at System.Net.Test.Common.Http3LoopbackStream.SendResponseHeadersAsync(Nullable`1 statusCode, IEnumerable`1 headers)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs(247,0): at System.Net.Test.Common.Http3LoopbackStream.SendResponseAsync(HttpStatusCode statusCode, IList`1 headers, String content, Boolean isFinal)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs(241,0): at System.Net.Test.Common.Http3LoopbackConnection.HandleRequestAsync(HttpStatusCode statusCode, IList`1 headers, String content)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs(93,0): at System.Net.Test.Common.Http3LoopbackServer.HandleRequestAsync(HttpStatusCode statusCode, IList`1 headers, String content)
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs(93,0): at System.Net.Test.Common.Http3LoopbackServer.HandleRequestAsync(HttpStatusCode statusCode, IList`1 headers, String content)
        /_/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs(128,0): at System.Net.Http.Functional.Tests.HttpClientHandler_MaxResponseHeadersLength_Test.<>c__DisplayClass5_0.<<ThresholdExceeded_ThrowsException>b__1>d.MoveNext()
        --- End of stack trace from previous location ---
        /_/src/libraries/Common/tests/System/Threading/Tasks/TaskTimeoutExtensions.cs(120,0): at System.Threading.Tasks.TaskTimeoutExtensions.GetRealException(Task task)
        --- End of stack trace from previous location ---
        /_/src/libraries/Common/tests/System/Threading/Tasks/TaskTimeoutExtensions.cs(90,0): at System.Threading.Tasks.TaskTimeoutExtensions.WhenAllOrAnyFailed(Task[] tasks)
        /_/src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs(38,0): at System.Net.Test.Common.LoopbackServerFactory.<>c__DisplayClass5_0.<<CreateClientAndServerAsync>b__0>d.MoveNext()
        --- End of stack trace from previous location ---
        /_/src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs(111,0): at System.Net.Test.Common.Http3LoopbackServerFactory.CreateServerAsync(Func`3 funcAsync, Int32 millisecondsTimeout, GenericLoopbackOptions options)
        /_/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs(96,0): at System.Net.Http.Functional.Tests.HttpClientHandler_MaxResponseHeadersLength_Test.ThresholdExceeded_ThrowsException(Nullable`1 maxResponseHeadersLength, Int32 headersLengthEstimate)
        --- End of stack trace from previous location ---
```

Cc @MihaZupan

### PR DESCRIPTION
**Pull request by wfurt**
This is mostly test problem. I saw occasional failures also on HTTP/1.1
```

System.Net.Http.Functional.Tests.SocketsHttpHandler_HttpClientHandler_MaxResponseHeadersLength_Http11.ThresholdExceeded_ThrowsException

      System.IO.IOException : Unable to write data to the transport connection: Broken pipe.
      ---- System.Net.Sockets.SocketException : Broken pipe
      Stack Trace:
        /home/furt/github/wfurt-runtime2/src/libraries/Common/tests/System/Net/Http/LoopbackServer.cs(739,0): at System.Net.Test.Common.LoopbackServer.Connection.SendResponseAsync(Byte[] response)
        /home/furt/github/wfurt-runtime2/src/libraries/Common/tests/System/Net/Http/LoopbackServer.cs(983,0): at System.Net.Test.Common.LoopbackServer.Connection.SendResponseBodyAsync(Byte[] content, Boolean isFinal)
        /home/furt/github/wfurt-runtime2/src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs(161,0): at System.Net.Test.Common.GenericLoopbackConnection.SendResponseBodyAsync(String content, Boolean isFinal)
```

now, there are some significant differences and reasons why HTTP/3 fails often:

https://github.com/dotnet/runtime/blob/4734ee0ae9055e722cbe6184e561cc9f29936482/src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs#L165-L169

and then on receiving side something like 
```c#
        (frameType, payloadLength) = await ReadFrameEnvelopeAsync(cancellationToken).ConfigureAwait(false);

        private async ValueTask ReadHeadersAsync(long headersLength, CancellationToken cancellationToken)
        {
            // TODO: this header budget is sent as SETTINGS_MAX_HEADER_LIST_SIZE, so it should not use frame payload but rather 32 bytes + uncompressed size per entry.
            // https://tools.ietf.org/html/draft-ietf-quic-http-24#section-4.1.1
            if (headersLength > _headerBudgetRemaining)
            {
                _stream.Abort(QuicAbortDirection.Read, (long)Http3ErrorCode.ExcessiveLoad);
                throw new HttpRequestException(SR.Format(SR.net_http_response_headers_exceeded_length, _connection.Pool.Settings.MaxResponseHeadersByteLength));
            }
```
we  get header length from the frame header and we can fail and Abort without even waiting for the headers (and sending task to complete)

The fix for this is to simply swallow transport errors. 

Now looking at it, it seems like we are aborting wrong side. It somewhat does not matter as we will fail and abort both sides immediately but that delivers wrong application code to the peer. 

Lastly I was trying to use tracing to figure out what is happening I finally succeeded but I had to improve it somewhat. 
The current log is way to crude and it does not include useful information. So I split the generic log to each event handler so we can log also some specific parts. With that following fragments was creation for solving this:

```
Sender:
        01:06:54.7569379[Info] thisOrContextObject: QuicStream#64628464, memberName: WriteAsync, message: [strm][0x7F3464004F80] Stream writing memory of '3' bytes while not completing writes.
        01:06:54.7569734[Info] thisOrContextObject: QuicStream#64628464, memberName: NativeCallback, message: [strm][0x7F3464004F80] Received event SEND_COMPLETE
        01:06:54.7569903[Info] thisOrContextObject: QuicStream#64628464, memberName: WriteAsync, message: [strm][0x7F3464004F80] Stream writing memory of '15400' bytes while not completing writes.
        01:06:54.7584017[Info] thisOrContextObject: QuicStream#64628464, memberName: NativeCallback, message: [strm][0x7F3464004F80] Received event IDEAL_SEND_BUFFER_SIZE
        01:06:54.7584457[Info] thisOrContextObject: QuicStream#64628464, memberName: NativeCallback, message: [strm][0x7F3464004F80] Received event PEER_RECEIVE_ABORTED
        01:06:54.7594386[Info] thisOrContextObject: QuicStream#64628464, memberName: NativeCallback, message: [strm][0x7F3464004F80] Received event SEND_COMPLETE

Receiver:
        01:40:10.3530872[Info] thisOrContextObject: QuicStream#41727345, memberName: NativeCallback, message: [strm][0x7FA554013E40] Received event RECEIVE
        01:40:10.3531404[Info] thisOrContextObject: QuicStream#41727345, memberName: HandleEventReceive, message: [strm][0x7FA554013E40] Received 1 buffers, Total count of 3. totalCopied=3
        01:40:10.3532103[HandlerMessage] poolId: 34868631, workerId: 49584532, requestId: 0, memberName: ReadFrameEnvelopeAsync, message: Received frame 1 of length 15400.
        01:40:10.3532604[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Read with 263.
        01:40:10.3546351[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Write with 258.
        01:40:10.3547886[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Read with 268.
        01:40:10.3588480[Info] thisOrContextObject: QuicStream#41727345, memberName: NativeCallback, message: [strm][0x7FA554013E40] Received event SHUTDOWN_COMPLETE
```

so it is clear we received 3 bytes out of promised 15400 and aborted immediately. Peer (e.g. LoopBack server) will get PEER_RECEIVE_ABORTED before completing write. 

fixes #73930

### CODE REVIEW

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (rzikm) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:393]:
```suggestion
```

File: src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs
Comment (rzikm) [inline src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs:93]: I suggest being more descriptive why are we ignoring these
```suggestion
                // Client can respond by closing/aborting the underlying stream while we are still sending the headers, ignore these exceptions 
```

File: src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs
Comment (rzikm) [inline src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs:143]: I suggest being more descriptive why are we ignoring these
```suggestion
                // Client can respond by closing/aborting the underlying stream while we are still sending the headers, ignore these exceptions 
```

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (rzikm) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:621]:
```suggestion

        return QUIC_STATUS_SUCCESS;
```

**Review by rzikm — APPROVED** [review]
LGTM, small nits

File: src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs
Comment (MihaZupan) [inline src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs:845]: With the previous behavior of closing the write direction, were things still eventually being cleaned up correctly?

**Review by MihaZupan — APPROVED** [review]
Thanks for catching the H3 issue

File: src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs
Comment (wfurt) [reply src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs:845]: Yes, you can sort of see it ion the posted logs. Frirst comes from the ReadHeadersAsync above. 

```
        01:40:10.3532604[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Read with 263.

        01:40:10.3546351[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Write with 258.
        01:40:10.3547886[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Read with 268.
```
I think this comes from HTTP3 error processing. This is easy to reproduce so I can look where this duplicate reset is coming from.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:394]: This should get moved before the actual msquic call, since we might decide not to abort either side based on its state. So the log message may be misleading.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:462]: Could you also cover `QuicConnection` and `QuicListener` for a consistency.
Also could you log the varying arguments in a similar manner, like:
```
NetEventSource.Info(this, $"{this} Received event START_COMPLETE with {nameof(Status)}={data.Status}, {nameof(PeerAccepted)}={data.PeerAccepted}");
```

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:495]:
```suggestion
            NetEventSource.Info(this, $"{this} Received event RECEIVE with {nameof(BufferCount)}={data.BufferCount}, {nameof(TotalBufferLength)}={data.TotalBufferLength} and {nameof(totalCopied)}={totalCopied}");
```

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:508]:
```suggestion
```

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:636]: `QuicConnection` and `QuicListener`.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:596]: Honestly, why don't we add `ToString` to the event data structure and log it from one place?

**Review by ManickaP — CHANGES_REQUESTED** [review]
Would it be possible to add `ToString`s to the event data structures instead of adding logging into each individual event handler?

Also this change should be done in `QuicConnection` and `QuicListener` as well, not just in `QuicStream`. Or we end up with inconsistent messy logging as we had before.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (rzikm) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:596]: That change would need to go to the generated interop stubs from MsQuic then, right?

Comment (ManickaP) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:596]: Not necessarily, all the structs are partial, so we can add it to a different file. Or just define it as extension/static method.

Comment (wfurt) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:596]: I don't think that would be beneficial. It may contains many field (we don't even use) and dumping everything is going to make logs hard to read IMHO. It will incur performance penalty and bring no value. 
If we feel strongly about this we can do it at at the Verbose level - we use it in Sockets for example to dump part of the actual data. 
I think using nameof() on interesting fields and consistent formatting may be the best

File: src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs
Comment (wfurt) [reply src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs]: updated.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (wfurt) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs]: updated.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (wfurt) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:461]: QuicListener did not have anything interesting IMHO so I left is as is. I updated `QuicConnection` with same pattern.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (wfurt) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs]: why? methods should be separated by line IMHO. So as empty line should follow closing brace.
I think we should do separate style pass on the file. @rzikm seems to have best editor setting....

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs:450]: This is not an Error

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:617]:
```suggestion
            NetEventSource.Info(this, $"{this} Received event {type}");
```

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:461]: At least it has a new connection. And this is also about consistency.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs:456]: If you insist on putting empty line in between every HandleXYZ then it should be done everywhere. I was intentional with this, I don't mind the change but it should be the same everywhere.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs:478]:
```suggestion
```

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs
Comment (ManickaP) [inline src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs:561]:
```suggestion
            NetEventSource.Info(this, $"{this} Received event {type}");
```

**Review by ManickaP — APPROVED** [review]
LGTM modulo comments.

I'm still not 100% convinced we should put the logging into every method instead of having `ToString` or some common way to do it, but I'll not hold this PR on it.

File: src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs
Comment (ManickaP) [reply src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs:596]: It would be us coding and thus controlling what goes in.

**Pull request merged**
